utils/migration-scripts/src/olympia-carthage/olympia-query-node/queries/queries.graphql

```diff
@@ -0,0 +1,40 @@
+fragment MembershipFields on Membership {
+  id
+  handle
+  metadata {
+    name
+    about
+    # We will only support avatarUri to bootstrap, since there will be no storage
+    # infra to upload data objects..
+    avatar {
+      ... on AvatarUri {
+        avatarUri
+      }
+    }
+    # Was this properly implemented on olympia? Is there any valuable info there?
```

Comment:
Yes I think the reason not many members have set this info is because the feature was disabled from pioneer probably because it was not working as expected. Only one external resource can be set as apposed to an array.

Reply:
yep confirmed by pioneer team.